schemas/rdf/examples/AAS_Reference_shortExample.ttl

```diff
@@ -22,34 +22,25 @@
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 
-<http://customer.com/aas/9175_7013_7091_9168> rdf:type <https://admin-shell.io/aas/3/0/RC01/AssetAdministrationShell> ;
-    <https://admin-shell.io/aas/3/0/RC01/Identifiable/identification> [
-       a <https://admin-shell.io/aas/3/0/RC01/Identifier> ;
-       <https://admin-shell.io/aas/3/0/RC01/Identifier/identifier> "http://customer.com/aas/9175_7013_7091_9168"^^xsd:string ;
-       <https://admin-shell.io/aas/3/0/RC01/Identifier/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri> ;
-    ] ;
-    <https://admin-shell.io/aas/3/0/RC01/Referable/idShort> "a9175_7013_7091_9168"^^xsd:string ;
-    <https://admin-shell.io/aas/3/0/RC01/AssetAdministrationShell/assetInformation> [
-        a <https://admin-shell.io/aas/3/0/RC01/AssetInformation> ;
-        <https://admin-shell.io/aas/3/0/RC01/AssetInformation/globalAssetId> [
-          a  <https://admin-shell.io/aas/3/0/RC01/Reference> ;
-            <https://admin-shell.io/aas/3/0/RC01/Reference/key> [
-              a <https://admin-shell.io/aas/3/0/RC01/Key> ;
-              <https://admin-shell.io/aas/3/0/RC01/Key/type> <https://admin-shell.io/aas/3/0/RC01/IdentifiableElements/Asset> ;
-              <https://admin-shell.io/aas/3/0/RC01/Key/value> "http://customer.com/assets/KHBVZJSQKIY"^^xsd:string ;
-              <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri>
+<http://customer.com/aas/9175_7013_7091_9168> rdf:type <https://admin-shell.io/aas/3/0/RC02/AssetAdministrationShell> ;
+    <https://admin-shell.io/aas/3/0/RC02/Identifiable/id> "http://customer.com/aas/9175_7013_7091_9168"^^xsd:string ;
+    <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "a9175_7013_7091_9168"^^xsd:string ;
+    <https://admin-shell.io/aas/3/0/RC02/AssetAdministrationShell/assetInformation> [
+        a <https://admin-shell.io/aas/3/0/RC02/AssetInformation> ;
+        <https://admin-shell.io/aas/3/0/RC02/AssetInformation/globalAssetId> [
```

Comment:
globalAssetId is a global reference, not a model reference

Reply:
Corrected.